spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsTransmitter.java

```diff
@@ -128,7 +131,7 @@ private void transmitToYggAdmin(eu.coatrack.api.Metric metricToTransmit) {
             */
 
         } catch (Exception e) {
-            log.error("Exception when communicating with CoatRack admin server", e);
+            log.error(String.format("An unexpected error occurred when your CoatRack gateway contacted the central CoatRack portal in order to send the statistics/metrics from this call: please assure that your CoatRack gateway has a network connection and can reach %s", adminServerUrl), e);
```

Comment:
General comment: We should probably unify the naming in all error messages that are referring to "CoatRack admin". Maybe replace it by "central CoatRack portal/server/application" everywhere? Not sure which of portal/server/application is best. To be discussed.